### DISCUSSION TITLE
Поддержка гипервизора Proxmox

### DISCUSSION THREAD

**Discussion by jorikfon** [mikopbx/Core#142] [Ideas (Идеи)]
Несколько раз уже проскакивало от клиентов про этот гипервизор.
https://qa.askozia.ru/5546/необходим-guest-agent-корректной-работы-гипервизоре-proxmox

**Comment by boffart** [comment]
ну так на этом гипервизоре АТС работает.
да, агента нет, его сборка может стать проблемой, будет занимать много места и потянет кучу зависимостей.

**Reply by jorikfon** [reply]
Может нам для разных гипервизоров модулей наделать просто? И основную систему не будет нагружать, и не надо думать лишнего, запустил модуль он в кроне подключил свой тулз и работает. Потом делай снапшоты, копируй машину сколько тебе надо в рамках облака.

**Reply by boffart** [reply]
наделать можно, только факт, что пока не выходит собрать агента... и тянет он много зависимостей, 
нужно также отработать такие "сборки в t2", чтобы оно не тянулось в core

**Comment by matveevandrey** [comment]
Текущая версия MikoPBX прекрасно работает на гипервизоре Proxmox. 
немного не хватает поддержки Qemu-agent-а, но не критично
Используем в нескольких инсталляциях

**Reply by napaster** [reply]
Агента на самом деле сильно не хватает, потому что любые команды для миграции, бекапа и прочего подобного посылает агент внутрь гостевой ос. Да тоже выключение  посылает агент, которого нету. Его прям очень и очень сильно не хватает.

**Comment by ivoryblade** [comment]
А еще бы починить мику чтобы она в lxc опять заработала на PVE - в какой то момент обе моих инсталляции перестали запускаться мики. Контейнер поднимается но никакие порты не открываются, даже веб морда.
